### ISSUE TITLE
SSI includes always passing to index.php

### ISSUE THREAD

**Issue by luizboaretto** [putyourlightson/craft-blitz#595]
I have SSI includes 

`<!--#include virtual="/_includes/?action=blitz/include/cached&index=2395729283" -->`

in the cache directory I have `'action=blitz%2Finclude%2Fcached&index=2395729283'`

Tested with nginx and apache

Apache config:
```
RewriteCond %{DOCUMENT_ROOT}/cache/blitz/%{HTTP_HOST}/%{REQUEST_URI}/%{QUERY_STRING}/index.html -s
RewriteCond %{REQUEST_METHOD} GET
RewriteCond %{QUERY_STRING} !token= [NC]
RewriteRule .* /cache/blitz/%{HTTP_HOST}/%{REQUEST_URI}/%{QUERY_STRING}/index.html [L]
```

nginx config:
```
#- Blitz cache rewrite
set $cache_path false;
if ($request_method = GET) {
    set $cache_path /cache/blitz/$host/$uri/$args/index.html;
}
if ($args ~ "token=") {
    set $cache_path false;
}

#- Send would-be 404 requests to the cache path or Craft
location / {
    ssi on;
    try_files $cache_path $uri $uri/ /index.php?$query_string;
}
```

**Comment by bencroker** [comment]
What is the question or issue?

**Comment by luizboaretto** [comment]
What is expected is that nginx or apache will serve the includes that are cached and not go through index.php

**Comment by bencroker** [comment]
This should work. Do you have an `_includes` folder containing each of the cached includes in the cache directory?

**Comment by luizboaretto** [comment]
For sure, as I mentioned in my first comment:

in the cache directory I have `'action=blitz%2Finclude%2Fcached&index=2395729283'` folder

```
web/cache/blitz/staging.portal.com/_includes/
├── action=blitz%2Finclude%2Fcached&index=2320311357
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=2395729283
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=2640481881
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=289657702
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=3023494763
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=3201798021
│   └── index.html
├── action=blitz%2Finclude%2Fcached&index=3891590775
│   └── index.html
└── action=blitz%2Finclude%2Fcached&index=703993243
    └── index.html

8 directories, 8 files
```

I think some issues with **slashes** and **%2F**

**Comment by luizboaretto** [comment]
if I change:

from:
`<!--#include virtual="/_includes/?action=blitz/include/cached&index=2395729283" -->`

to:
`<!--#include virtual="/_includes/?action=blitz%2Finclude%2Fcached&index=2395729283" -->`

**works!**

**Comment by bencroker** [comment]
And you’re running the latest version of Blitz? I believe this was addressed in [4.6.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.6.0).

**Comment by luizboaretto** [comment]
Blitz 4.9.3
Craft Pro 4.5.13
PHP 8.2.11

And tested on
Apache/2.4.58
nginx/1.18.0

**Comment by luizboaretto** [comment]
> And you’re running the latest version of Blitz? I believe this was addressed in [4.6.0](https://github.com/putyourlightson/craft-blitz/releases/tag/4.6.0).

I think this the problem... you are not encoding the url and on the server it is encoded. I understand that the webserver should know this, but something is wrong. I don't know if it's a lack of configuration on my side or if it would be best to normalize this

**Comment by bencroker** [comment]
It sounds like the folders in the `_includes` directory might be leftovers from before the 4.6.0 release. Can you delete the `_includes` directory and test whether they are recreated without encoded slashes?

**Comment by luizboaretto** [comment]
I started this project in November... this is a new fresh server and clean database, etc... 

I'm pretty sure I'm already way ahead of 4.6.0

**Comment by luizboaretto** [comment]
I cleared all possible cache, but it's still the same thing

**Comment by bencroker** [comment]
I’m wondering if the change in 4.6.0 was redundant, after all. 

Can you please install and test this bug fix, clearing the cache first, by running `composer require "putyourlightson/craft-blitz:dev-bugfix/undo-decode-slashes as 4.9.4"`?

**Comment by luizboaretto** [comment]
**It works!** 🥇

**Comment by bencroker** [comment]
Changed and released in [4.9.4](https://github.com/putyourlightson/craft-blitz/releases/tag/4.9.4), thanks for reporting this.
